plugins/collections/get.js

```diff
@@ -41,8 +41,10 @@ function getPipelinePRInfo(pipeline) {
 
             prJobsBuilds.forEach(jobBuilds => {
                 // Check the first element (last build) if pr job failed
-                if (jobBuilds[0].status === 'FAILURE') {
-                    prs.failing += 1;
+                if (jobBuilds[0] !== undefined) {
```

Comment:
`if (jobBuilds && jobBuilds[0])` ? of if we want to be semantically correct, `if (Array.isArray(jobBuilds) && typeof jobBuilds[0] === 'object')`

Reply:
I got it. 
I'll fix the check process.